metadoc-js/src/main/scala/metadoc/ScalaDocumentSymbolProvider.scala

```diff
@@ -30,21 +30,21 @@ class ScalaDocumentSymbolProvider(index: MetadocSemanticdbIndex)
   }
 
   override def provideDocumentSymbols(
-      model: IReadOnlyModel,
+      model: ITextModel,
       token: CancellationToken
   ) = {
     for {
       Some(doc) <- index.semanticdb(model.uri.path)
     } yield {
       val symbols = getDocumentSymbols(doc).map {
         case DocumentSymbol(denotation, kind, definition) =>
-          new SymbolInformation(
-            name = denotation.displayName,
-            // TODO: pretty print `.tpe`: https://github.com/scalameta/scalameta/issues/1479
-            containerName = denotation.symbol,
-            kind = kind,
-            location = model.resolveLocation(definition)
-          )
+          val symbol = jsObject[SymbolInformation]
+          symbol.name = denotation.displayName
+          // TODO: pretty print `.tpe`: https://github.com/scalameta/scalameta/issues/1479
```

Comment:
The issue has since been closed. Not sure if it is relevant anymore.

Reply:
I've updated the comment to link to #99